src/index.html

```diff
@@ -11,6 +11,82 @@
+    <form
+      class="form"
+      method="get"
+      action="/"
+      data-qa="big"
+    >
+      <label class="search">
+        <svg class="icon" 
+          width="19" 
+          height="19" 
+          viewBox="0 0 19 19" 
+          fill="none" 
+          xmlns="http://www.w3.org/2000/svg">
+          <path
+            d="M13.8401 13.0856L13.5339 13.4375L13.8638 13.7673L18.4821 
+            18.3846C18.4925 18.3949 18.5 18.4106 18.5 18.4313C18.5 18.4519 
+            18.4925 18.4676 18.4821 18.4779L18.4821 18.4778L18.4755 
+            18.4846C18.4677 18.4928 18.4538 18.5 18.4354 18.5C18.4318 18.5 
+            18.4242 18.4991 18.4143 18.4951C18.4047 18.4911 18.3958 18.4852 
+            18.3886 18.4779L13.7702 13.8606L13.4405 13.5309L13.0886 
+            13.8369C11.7516 14.9996 10.0088 15.7027 8.10313 15.7027C3.91254 
+            15.7027 0.5 12.2907 0.5 8.10133C0.5 3.91178 3.9085 0.5 8.10313 
+            0.5C12.2939 0.5 15.7063 3.9079 15.7063 8.10133C15.7063 10.0065 
+            15.0031 11.7489 13.8401 13.0856ZM8.09891 0.637472C3.98431 0.637472 
+            0.633511 3.98276 0.633511 8.10133C0.633511 12.2195 3.98391 15.5694 
+            8.09891 15.5694C12.2185 15.5694 15.5643 12.2149 15.5643 
+            8.10133C15.5643 3.98736 12.2181 0.637472 8.09891 0.637472Z"
+            fill="#687C94" stroke="#687C94"
+          >
+        </svg> 
+        <input
+          class="input" 
+          type="text"
+          name="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </label>  
+    </form>
+
+    <form
+      class="form form--small" 
+      method="get" 
+      action="/"
+      data-qa="small"
+    >
+      <label class="search search--small">
+        <svg class="icon icon--small" 
+          width="11"
+          height="11"
+          viewBox="0 0 19 19"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg">
+          <path
+            d="M13.8401 13.0856L13.5339 13.4375L13.8638 13.7673L18.4821 
+            18.3846C18.4925 18.3949 18.5 18.4106 18.5 18.4313C18.5 18.4519 
+            18.4925 18.4676 18.4821 18.4779L18.4821 18.4778L18.4755 18.4846C18.4677 
+            18.4928 18.4538 18.5 18.4354 18.5C18.4318 18.5 18.4242 18.4991 
+            18.4143 18.4951C18.4047 18.4911 18.3958 18.4852 18.3886 18.4779L13.7702 
+            13.8606L13.4405 13.5309L13.0886 13.8369C11.7516 14.9996 10.0088 15.7027 
+            8.10313 15.7027C3.91254 15.7027 0.5 12.2907 0.5 8.10133C0.5 3.91178 
+            3.9085 0.5 8.10313 0.5C12.2939 0.5 15.7063 3.9079 15.7063 8.10133C15.7063 
+            10.0065 15.0031 11.7489 13.8401 13.0856ZM8.09891 0.637472C3.98431 0.637472 
+            0.633511 3.98276 0.633511 8.10133C0.633511 12.2195 3.98391 15.5694 8.09891 
+            15.5694C12.2185 15.5694 15.5643 12.2149 15.5643 8.10133C15.5643 3.98736 
+            12.2181 0.637472 8.09891 0.637472Z"
+            fill="#687C94" stroke="#687C94"
+          >
+        </svg> 
```

Comment:
```suggestion
         <img 
          src="images/Search.svg" 
          alt="search" 
          class="icon icon--small"
        >
```

Fix previous comment please 
![image](https://user-images.githubusercontent.com/70009205/218537623-d724e7d6-d28f-43d1-a492-c8e5d90ba2b7.png)